page.php

```diff
@@ -82,6 +82,11 @@
 if ( post_password_required( $post->ID ) ) {
```

Comment:
It could be better to add a check in html-header template to check if a post is rendered and if it is password protected and then render the meta tags.

something like
`{% if (post and fn('post_password_required', post)) %}`